docs/configs/fluence.md

```diff
@@ -16,7 +16,7 @@ Defines Fluence Project, most importantly - what exactly you want to deploy and
 | `deals`                | [object](#deals)            | No       | A map of objects with worker names as keys, each object defines a deal                                                                                                                                                                                                                                                                                                                                                               |
 | `defaultSecretKeyName` | string                      | No       | Secret key with this name will be used by default by js-client inside CLI to run Aqua code                                                                                                                                                                                                                                                                                                                                           |
 | `dependencies`         | [object](#dependencies)     | No       | (For advanced users) Overrides for the project dependencies                                                                                                                                                                                                                                                                                                                                                                          |
-| `hosts`                | [object](#hosts)            | No       | A map of objects with worker names as keys, each object defines a list of peer IDs to host the worker on                                                                                                                                                                                                                                                                                                                             |
+| `hosts`                | [object](#hosts)            | No       | A map of objects with worker names as keys, each object defines a list of peer IDs to host the worker on. If you want to deploy your services to specific peerIds. Intended to be used by providers to deploy directly without using the blockchain                                                                                                                                                                                  |
```

Comment:
```suggestion
| `hosts`                | [object](#hosts)            | No       | A map of objects with worker names as keys, each object defines a list of peer IDs to host the worker on. Intended to be used by providers to deploy directly without using the blockchain                                                                                                                                                                                  |
```
Probably redundant sentence. You already said that "list of peer IDs to host the worker on".